Library/ViewModels/NoShippingPledgeViewModelTests.swift

```diff
@@ -5226,4 +5229,50 @@ final class NoShippingPledgeViewModelTests: TestCase {
       )
     }
   }
+  
+  func testShowPledgeOverTimeUI_True() {
+    withEnvironment(currentUser: nil) {
+      let project = Project.template
+      |> Project.lens.isPledgeOverTimeAllowed .~ true
+      let reward = Reward.template
+
+      let data = PledgeViewData(
+        project: project,
+        rewards: [reward],
+        selectedShippingRule: shippingRule,
+        selectedQuantities: [reward.id: 1],
+        selectedLocationId: nil,
+        refTag: .projectPage,
+        context: .pledge
+      )
+
+      self.vm.inputs.configure(with: data)
+      self.vm.inputs.viewDidLoad()
+      
+      self.showPledgeOverTimeUI.assertValues([true])
+    }
+  }
+  
+  func testShowPledgeOverTimeUI_False() {
+    withEnvironment(currentUser: nil) {
```

Comment:
nit: same comment as the one above about not needing to explicitly set currentUser to nil here